.github/workflows/go.yml

```diff
@@ -53,6 +53,6 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.48.0
+          version: latest
```

Comment:
are you sure about that? it could lead to the "make PR, go sleep, wake up, PR is outdated because of linter release despite the same code"

Reply:
Fixed